services/src/main/java/org/fao/geonet/api/users/UsersApi.java

```diff
@@ -263,6 +270,52 @@ public User getUser(
         return new ResponseEntity(HttpStatus.NO_CONTENT);
     }
 
+    @ApiOperation(
+        value = "Check if a user property already exist",
+        notes = "",
+        authorizations = {
+            @Authorization(value = "basicAuth")
+        },
+        nickname = "checkUserPropertyExist")
+    @RequestMapping(
+        value = "/properties/{property}",
+        method = RequestMethod.GET
+    )
+    @ResponseStatus(value = HttpStatus.OK)
+    @PreAuthorize("hasRole('UserAdmin')")
+    @ApiResponses(value = {
+        @ApiResponse(code = 200, message = "Property does not exist."),
+        @ApiResponse(code = 404, message = "A property with that value already exist."),
+        @ApiResponse(code = 403, message = ApiParams.API_RESPONSE_NOT_ALLOWED_ONLY_USER_ADMIN)
+    })
+    public ResponseEntity<HttpStatus> checkHarvesterPropertyExist(
```

Comment:
The name of the method I guess should be something like `checkUSerPropertyExist`?